internal/locate/region_request.go

```diff
@@ -2302,3 +2310,34 @@ func (s *staleReadMetricsCollector) onResp(tp tikvrpc.CmdType, resp *tikvrpc.Res
 		metrics.StaleReadRemoteInBytes.Add(float64(size))
 	}
 }
+
+func getReadType(req *tikvrpc.Request) string {
+	if req.StaleRead {
+		return "stale"
+	}
+	if req.ReplicaRead {
+		return "replica"
+	}
+	return "leader"
+}
+
+func patchRequestSource(req *tikvrpc.Request, source, readType string, retryTimes int) {
+	var sb strings.Builder
+	sb.WriteString(source)
+	sb.WriteByte('-')
```

Comment:
Both `-` and `_` exist here, how about using only one for unification?

Reply:
`-` devides request source and read type, `_` is the inner separator of each one.